vitrina/projects/forms.py

```diff
@@ -45,3 +47,61 @@ def __init__(self, *args, **kwargs):
             Field("image"),
             Submit("submit", button, css_class="button is-primary"),
         )
+
+class ClientCreateForm(ModelForm):
+    name = CharField(label=_("Pavadinimas"), help_text=_("Kliento pavadinimas"))
+
+    class Meta:
+        model = UseCaseClient
+        fields = ["name"]
+
+    def __init__(self, *args, **kwargs):
+        super().__init__(*args, **kwargs)
+        project_instance = self.instance if self.instance and self.instance.pk else None
+        button = _("Redaguoti") if project_instance else _("Sukurti")
+        self.helper = FormHelper()
+        self.helper.attrs["novalidate"] = ""
+        self.helper.form_id = "client-form"
+        self.helper.layout = Layout(
+            Field("name", placeholder=_("Pavadinimas")),
+            Submit("submit", button, css_class="button is-primary"),
+        )
+
+
+class ClientScopeCreateForm(ModelForm):
+    scope = ModelChoiceField(
+        queryset=AgreementScope.objects.none(),
+        label=_("Leidimas"),
+        help_text=_("Pasirinkite leidimą"),
+    )
+
+    class Meta:
+        model = UseCaseClientScope
+        fields = ["scope"]
+
+    def __init__(self, *args, available_scopes=None, use_case_client=None, **kwargs):
+        super().__init__(*args, **kwargs)
+        if available_scopes is not None and use_case_client is not None:
```

Comment:
Wouldn't this suffice?

```suggestion
        if available_scopes and use_case_client:
```